AEPServices/Sources/ui/fullscreen/FullscreenMessage+FrameCalculation.swift

```diff
@@ -119,13 +119,17 @@
         /// width in settings represents a percentage of the screen.
         /// e.g. - 80 = 80% of the screen width.
```

Comment:
Are we changing the definition of width now to not be a percentage if maxWidth is set? If so, we should update the definition here as its a bit confusing.

Reply:
no. this comment is referring to the value in settings.  i'll add a comment that explains the broader purpose of the calculated variable for clarity (and for height as well)